src/freertos_drivers/esp32/Esp32WiFiManager.cxx

```diff
@@ -50,15 +50,16 @@
 
 // ESP-IDF v4+ has a slightly different directory structure to previous
 // versions.
-#ifdef ESP_IDF_VERSION_MAJOR
```

Comment:
plesae putthis change into a separate PR as it has nothing to do with the make unique.

Reply:
Ok, will split but it won't compile without it very soon.